WebRoot/img/zimbra/ImgAttachClip.svg

```diff
@@ -0,0 +1,3 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
Please see: https://github.com/Zimbra/zm-web-client/pull/6/files#diff-c222f38db9974ef443747ae87b6c374cR1 as reference to remove xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32"